go/vt/discovery/healthcheck.go

```diff
@@ -334,11 +355,89 @@ func (hc *HealthCheckImpl) deleteEndPointFromTargetProtected(target *pbq.Target,
 		if topo.EndPointEquality(ep, endPoint) {
 			epList = append(epList[:i], epList[i+1:]...)
 			ttMap[target.TabletType] = epList
+			hcConnCounters.Add([]string{target.Keyspace, target.Shard, strings.ToLower(target.TabletType.String())}, -1)
 			return
 		}
 	}
 }
 
+// EndPointsCacheStatus is the current endpoints for a cell/target.
+type EndPointsCacheStatus struct {
+	Cell           string
+	Target         *pbq.Target
+	EndPointsStats []*EndPointStats
+}
+
+// StatusAsHTML returns an HTML version of the status.
```

Comment:
How about add a TODO to change this status to reflect the End-to-End information about the endpoints. E.g.

bns_of_endpoint1 querys/errors